src/components/ha-tabs.ts

```diff
@@ -34,6 +34,9 @@ export class HaTabs extends PaperTabs {
 
       superStyle!.appendChild(
         document.createTextNode(`
+          :host {
+            padding-top: .5px;
```

Comment:
Spot the difference 🔎 😄

Reply:
Oh trust me, it's visible. 😄

Without the .5px there is a slight line below the tab indicator, with a full 1px it overlaps the header bottom.